Odata-docs/odatalib/v6/read-payload-v6.md

```diff
@@ -1,9 +1,8 @@
 ---
-title: " Read OData Payload"
-description: "Read OData payload using OData Core APIs"
-
+title: "Read OData Payload"
+description: Describes how to read OData payloads using OData Core APIs.
```

Comment:
@dereklh7 Does this mean that the quotation marks are optional?

Reply:
@gathogojr Quotation marks are indeed optional for descriptions and titles.